### ISSUE TITLE
Safari: Wrong focus indicator on Input

### ISSUE THREAD

**Issue by sheff146** [microsoft/fluentui#12404]
### Environment Information

- **Package version(s)**: 0.47
- **Browser and OS versions**: Safari, MacOS

Safari adds a weird focus indicator on Input and Dropdown (searchable).
![image](https://user-images.githubusercontent.com/4189399/77415745-3268ca00-6dc3-11ea-89fd-299a58835b2e.png)
![image](https://user-images.githubusercontent.com/4189399/77415834-51fff280-6dc3-11ea-8a2f-1d4888bd0089.png)

#### Actual behavior:
Safari adds a blue ring

#### Expected behavior:
Focus indicator should be defined by a theme

### Priorities and help requested:

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: Fluent UI doc site

**Comment by sheff146** [comment]
This is an issue from the old Fluent UI repository https://github.com/microsoft/fluent-ui-react/issues/2376

**Comment by aneeshack4** [comment]
@sheff146 thanks for reporting and opening a PR to address this!
